drizzlepac/astrodrizzle.py

```diff
@@ -159,6 +159,23 @@ def run(configobj, wcsmap=None):
         def_logname = None
         return
 
+    # validate input values for keep_file parameter
+    clean_files = ['blotImage','crmaskImage','finalMask',
+                    'staticMask','singleDrizMask','outSky',
+                    'outSContext','outSWeight','outSingle',
+                    'outMedian','dqmask','tmpmask',
+                    'skyMatchMask']
+    invalid_entries = []
+    for kfile in keep_files:
+        if kfile not in clean_files:
+            invalid_entries.append(kfile)
```

Comment:
This is an ideal application for sets:
```python
invalid_entries  = list(set(keep_files).difference(clean_files))
```

Reply:
Implemented in https://github.com/spacetelescope/drizzlepac/pull/301/commits/95f65b33f237cc3313feaf05492717f6714443ef